craft_parts/plugins/go_use_plugin.py

```diff
@@ -100,5 +100,5 @@ def get_build_commands(self) -> list[str]:
                 )
 
         return [
-            f"go work use {self._part_info.part_src_dir}",
+            f"go work use {self._part_info.part_src_subdir}",
```

Comment:
Can you explain to me why these are the source dir/subdir and not the build dir/subdir? It's not a blocker for this, it's just that the latter makes more sense to me.

Reply:
in the before times, src was not copied of to build